packages/garbo/src/resources/candyMap.ts

```diff
@@ -0,0 +1,123 @@
+import { Outfit } from "grimoire-kolmafia";
+import { $familiar, $item, get, getSaleValue, have, maxBy, sum } from "libram";
+import {
+  canEquip,
+  getOutfits,
+  holiday,
+  mallPrice,
+  outfitPieces,
+  outfitTreats,
+  print,
+  runChoice,
+  toItem,
+  use,
+  visitUrl,
+} from "kolmafia";
+import { GarboTask } from "../tasks/engine";
+import { garboValue } from "../garboValue";
+import { acquire } from "../acquire";
+import { GarboStrategy, Macro } from "../combat";
+
+const HOUSE_NUMBERS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+
+let blockHtml = "";
+
+function getBlockHtml(): string {
+  blockHtml ||= visitUrl("place.php?whichplace=town&action=town_trickortreat");
+  return blockHtml;
+}
+
+function treatValue(outfit: string): number {
+  return sum(
+    Object.entries(outfitTreats(outfit)),
+    ([candyName, probability]) => probability * garboValue(toItem(candyName)),
+  );
+}
+
+export function getTreatOutfit(): string {
+  const availableOutfits = getOutfits().filter((name) =>
+    outfitPieces(name).every((piece) => canEquip(piece)),
+  );
+  if (!availableOutfits.length) {
+    print("You don't seem to actually have any outfits available, my friend!");
```

Comment:
```suggestion
    print("You don't seem to actually have any outfits available, my friend!");
```
This message should mention trick-or-treat, otherwise users won't know what it refers to.